Userland/Libraries/LibWeb/HTML/Window.h

```diff
@@ -141,6 +147,11 @@ class Window final
     RefPtr<DOM::Event> m_current_event;
 
     HashMap<i32, NonnullRefPtr<RequestAnimationFrameCallback>> m_request_animation_frame_callbacks;
+
+    // https://w3c.github.io/requestidlecallback/#dfn-list-of-idle-request-callbacks
+    NonnullRefPtrVector<IdleCallback> m_idle_request_callbacks;
+    NonnullRefPtrVector<IdleCallback> m_runnable_idle_callbacks;
```

Comment:
Likewise

https://w3c.github.io/requestidlecallback/#dfn-list-of-runnable-idle-callbacks
https://w3c.github.io/requestidlecallback/#dfn-idle-callback-identifier